src/components/DeviceDetails/index.js

```diff
@@ -67,24 +65,25 @@ class DeviceDetails extends React.Component {
           <div className="device-details__table">
             <div className="device-details__header-row device-details__row">
               <div className="device-details__row-label">Apparaat</div>
-              <div className="device-details__row-element">
-                {this.props.device.name}
-              </div>
+              <div className="device-details__row-element">{this.props.device.name}</div>
             </div>
             <div className="device-details__row">
               <div className="device-details__row-label">Categorie</div>
-              <div className="device-details__row-element">
-                {getMarkerCategory(this.props.device).name}
-              </div>
+              <div className="device-details__row-element">{this.props.device.category}</div>
               {TypesButton}
             </div>
-            {this.props.device.types && (
+            {this.props.device.soort && (
               <div className="device-details__row">
                 <div className="device-details__row-label">Type</div>
-                <div className="device-details__row-element">
-                  {(this.props.device.types.length &&
-                    this.props.device.types[0].name) ||
-                    'Onbekend'}
+                <div className="device-details__row-element">{this.props.device.soort || 'Onbekend'}</div>
+              </div>
+            )}
+            {this.props.device.privacy && (
+              <div className="device-details__row">
+                <div className="device-details__row-label">Privacyverklaring</div>
+                <div className="device-details__row-element"><a href={this.props.device.privacy} target="_blank">
```

Comment:
You should add `rel="noopener noreferrer"` here. See https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/ for reference.